servers/graphql-kotlin-spring-server/src/main/kotlin/com/expediagroup/graphql/server/spring/subscriptions/ApolloSubscriptionSessionState.kt

```diff
@@ -31,43 +30,24 @@ internal class ApolloSubscriptionSessionState {
     // Operations are saved by web socket session id, then operation id
     internal val activeOperations = ConcurrentHashMap<String, ConcurrentHashMap<String, Subscription>>()
 
-    // The context is saved by web socket session id
-    private val cachedContext = ConcurrentHashMap<String, GraphQLContext>()
-
     // The graphQL context is saved by web socket session id
-    private val cachedGraphQLContext = ConcurrentHashMap<String, Map<*, Any>>()
+    private val cachedGraphQLContext = ConcurrentHashMap<String, Map<*, Any?>>()
 
     /**
      * Save the context created from the factory and possibly updated in the onConnect hook.
      * This allows us to include some intial state to be used when handling all the messages.
      * This will be removed in [terminateSession].
      */
-    fun saveContext(session: WebSocketSession, graphQLContext: GraphQLContext?) {
-        if (graphQLContext != null) {
-            cachedContext[session.id] = graphQLContext
-        }
-    }
-
-    /**
-     * Save the context created from the factory and possibly updated in the onConnect hook.
-     * This allows us to include some intial state to be used when handling all the messages.
-     * This will be removed in [terminateSession].
-     */
-    fun saveContextMap(session: WebSocketSession, graphQLContext: Map<*, Any>?) {
+    fun saveContextMap(session: WebSocketSession, graphQLContext: Map<*, Any?>?) {
```

Comment:
If we are making the breaking change do we want to call it `saveGraphQLContext` to match conventions?

Reply:
Yeah was just trying to get it compile first.